deploy_to_openshift.sh

```diff
@@ -192,7 +200,8 @@ then
   gologfilewatcher_image="docker.io/cognetive/go-log-file-watcher-with-symlink-support-v0"
   fluentbit_image="fluent/fluent-bit:1.7-debug"
   fluentbit_conf="conf/collector/fluentbit/fluentbit-clf.conf"
-  vector_image="quay.io/dciancio/vector"
+  #vector_image="quay.io/dciancio/vector"
+  vector_image=quay.io/openshift-logging/vector:0.14.1
```

Comment:
Can you please change `vector_image` to the latest stable version of vector?

Reply:
When I use this image, I don't see `vector_component_received_events_total` metrics. Can you please change it to `timberio/vector:0.20.0-debian`?